ykman/cli/openpgp.py

```diff
@@ -107,6 +108,16 @@ def info(ctx):
     controller = ctx.obj["controller"]
     click.echo("\n".join(pretty_print(get_openpgp_info(controller))))
 
+    list: List[Tuple[str, str]] = []
+    list.append(("Signature pin policy", controller.get_signature_pin_policy()))
+
+    column_left_lengths = [len(x[0]) for x in list]
+    max_column_left = max(column_left_lengths)
+
+    output = map(lambda x: f"{x[0].ljust(max_column_left)}\t{x[1]}", list)
+
+    click.echo("\n".join(output))
```

Comment:
I think it would make sense for this to be added to the output of `get_openpgp_info` (in ykman/openpgp.py) and just get printed here as the result of that. There you also won't have to do any formatting since `pretty_print` will take care of that.

The word "PIN" should be capitalized.